### ISSUE TITLE
The ckpt has a social decoder, but the code does not.

### ISSUE THREAD

**Issue by xushilin1** [decisionforce/mmTransformer#20]
Thanks for your code. I encountered several questions when reproducing this paper.  
I hope you can help me solve these questions.
- Code did not have a social decoder, but the ckpt you provided has. I'm interested in how you trained this ckpt.
- What is the classification loss you choose for six trajectory proposals? CrossEntropy or KL and what is the target ?
- What is the loss weight for reg loss and cls loss?
Thanks for your project again and I really hope to get your reply.

**Comment by DinghaoXi** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我已经收到邮件，会尽快给您回复。by小溪
